### PR TITLE
Automated PR: Cookstyle Changes

### PR DESCRIPTION
**Pull request by xorimabot**
Hey!
I ran Cookstyle 6.6.9 against this repo and here are the results.
This repo was selected due to the topics of chef-cookbook

### Issues found and resolved with libraries/helpers.rb

 - 120:44 refactor: `ChefCorrectness/InvalidPlatformInCase` - Use valid platform values in case statements.
 - 154:24 refactor: `ChefCorrectness/InvalidPlatformFamilyInCase` - Use valid platform family values in case statements.
 - 161:14 refactor: `ChefCorrectness/InvalidPlatformFamilyInCase` - Use valid platform family values in case statements.
 - 193:44 refactor: `ChefCorrectness/InvalidPlatformInCase` - Use valid platform values in case statements.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: libraries/helpers.rb [hunk 1]
```diff
@@ -151,14 +151,14 @@ def dhcpd_lease_file(ip_version)
...
-        when 'amazon', 'centos', 'fedora', 'rhel'
+        when 'amazon' 'fedora', 'rhel'
           {
             'owner' => 'dhcpd',
             'group' => 'dhcpd',
             'mode' => '0644',
             'action' => :create_if_missing,
           }
-        when 'ubuntu'
+        when 'debian'
```
Comment (tas50) [inline libraries/helpers.rb:161]: This one can get nuked entirely

File: libraries/helpers.rb [hunk 2]
```diff
@@ -190,7 +190,7 @@ def dhcpd_systemd_unit_content(ip_version, config_file)
         dhcp6 = ip_version.eql?(:ipv6)
 
         case node['platform']
-        when 'amazon', 'centos', 'fedora', 'rhel'
+        when 'amazon', 'centos', 'fedora', 'redhat'
```
Comment (tas50) [inline libraries/helpers.rb:193]: This should potentially get converted to a platform_family check depending on what's below line 196

**Comment by Xorima** [comment]
@bmhughes would you be able to fix the issues highlighted by cookstyle in this pr?

File: libraries/helpers.rb (hunk 1 above)
Comment (bmhughes) [reply libraries/helpers.rb]: That's fat fingers on my part, I've fixed it.

File: libraries/helpers.rb (hunk 2 above)
Comment (bmhughes) [reply libraries/helpers.rb:193]: Unless there's a way to distinguish between Debian and Ubuntu in a platform_family then it's going to have to stay as a family check. I agree having to specify all the RH based platforms looks untidy.

**Comment by bmhughes** [comment]
Done, I've fixed up the duplicate platform.

**Pull request merged**